lib/components/artist/biography.js

```diff
@@ -21,7 +21,7 @@ class Biography extends React.Component {
 
   render() {
     const artist = this.props.artist;
-    if (artist.blurb.length === 0 && artist.bio.length === 0) { return null; }
+    if (!artist.blurb && !artist.bio) { return null; }
```

Comment:
Hmm I would hope so too. I guess it depends on metaphysics but it seems much more likely that it would be `null` instead of an empty string so this solution makes sense to me.